app/src/main/java/com/mashup/mobalmobal/ui/createdonation/CreateDonationFragment.kt

```diff
@@ -224,19 +227,26 @@ class CreateDonationFragment : BaseViewBindingFragment<FragmentCreateDonationBin
             if (viewId == START_DATE_TIME_PICKER) {
                 createDonationViewModel.setCreateDonationStartDate(calendar.timeInMillis)
                 binding.createDonationStartDateInput.setText(formattedDateTime)
-                goDownAnimation(
-                    3,
-                    binding.createDonationDueDateInputLayout,
-                    null
-                )
+                if (!binding.createDonationProductImageView.isVisible) {
+                    goDownAnimation(
+                        3,
+                        binding.createDonationDueDateInputLayout,
+                        binding.createDonationDueDateInput
+                    )
+                }
+
             } else {
                 createDonationViewModel.setCreateDonationDueDate(calendar.timeInMillis)
                 binding.createDonationDueDateInput.setText(formattedDateTime)
-                goDownAnimation(
-                    4,
-                    binding.createDonationProductImageView,
-                    null
-                )
+
+                if (!binding.createDonationProductImageView.isVisible) {
+                    goDownAnimation(
+                        4,
+                        binding.createDonationProductImageView,
+                        null
+                    )
```

Comment:
필수는 아닌데 앞으로 인자 적은게 1줄 이상 길어질 경우 앞에 parameter name 붙여주세요!
```suggestion
                    goDownAnimation(
                        viewIndex = 4,
                        nextViewLayout = binding.createDonationProductImageView,
                        nextView = null
                    )
```

Reply:
앗 알겠습니당.